docs/data_source/catalog/iceberg_catalog.md

```diff
@@ -953,6 +953,14 @@ Currently StarRocks only supports [identity transforms](https://iceberg.apache.o
 
 You can specify the table attributes in the `"key" = "value"` format in `properties`. See [Iceberg table attributes](https://iceberg.apache.org/docs/latest/configuration/).
 
+The following table describes a few key properties.
+
+| **Property**      | **Description**                                              |
+| ----------------- | ------------------------------------------------------------ |
+| location          | The file path in which you want to create the Iceberg table. If you do not specify this property, the table defaults to the file path of the database to which the table belongs. |
```

Comment:
User must specify the location using glue if not specified when creating the table .